misk-clustering-dynamodb/src/main/kotlin/misk/clustering/dynamo/DynamoClusterModule.kt

```diff
@@ -0,0 +1,33 @@
+package misk.clustering.dynamo
+
+import com.google.inject.Provides
+import jakarta.inject.Qualifier
+import jakarta.inject.Singleton
+import misk.ServiceModule
+import misk.clustering.Cluster
+import misk.inject.KAbstractModule
+import misk.tasks.RepeatedTaskQueue
+import misk.tasks.RepeatedTaskQueueFactory
+import java.util.UUID
+
+class DynamoClusterModule : KAbstractModule() {
+  override fun configure() {
+    val dynamoCluster = DynamoCluster(System.getenv("MY_POD_NAME") ?: UUID.randomUUID().toString())
```

Comment:
I don't think we should use `System.getenv("MY_POD_NAME")` here, as that is no longer guaranteed to be unique? Maybe just `UUID` always?

Reply:
what're we doing for logs, metrics, etc? until now we've always operated with pod name as the ID, are we changing that for datadog stuff?

if i see, for example, a high cpu usage node in datadog how do i determine if it owns many leases?

Reply:
i'm fine with uuid here just curious in general mostly

Reply:
Ah, I commented later that we should have the pod_name as additional nonfunctional metadata in the table? For metrics and logs